app/services/provider_interface/send_candidate_withdrawn_on_request_email.rb

```diff
@@ -9,25 +9,7 @@ def initialize(application_choice:)
     end
 
     def call
-      if application_choice.continuous_applications?
-        CandidateMailer.application_withdrawn_on_request(application_choice).deliver_later
-      else
-        pre_continuous_applications_withdrawn_mailers
-      end
-    end
-
-  private
-
-    def pre_continuous_applications_withdrawn_mailers
-      if candidate_applications.all?(&:withdrawn?)
-        CandidateMailer.application_withdrawn_on_request_all_applications_withdrawn(application_choice).deliver_later
-      elsif applications_with_offer_and_awaiting_decision?
```

Comment:
These definitions can also be removed once [this PR ](https://github.com/DFE-Digital/apply-for-teacher-training/pull/8875) is merged.